tests/test_git_push.rs

```diff
@@ -187,10 +198,22 @@ fn test_git_push_changes() {
     }
     let stdout = test_env.jj_cmd_success(&workspace_root, &["git", "push", "--change", "@"]);
     insta::assert_snapshot!(replace_changeid(&stdout), @r###"
-    Creating branch push-<CHANGE_ID> for revision @
+    Creating branch push-<CHANGE_ID> for revision <CHANGE_ID>
```

Comment:
This change seems undesired. TODO: Figure out what happened.

Reply:
Sure, we can do that. Do we want to? Other commands don't seem to do it. Though, in this specific case, it does give a bit of potentially useful information.

Reply:
I don't care much about it. I was just trying to answer your question. Feel free (as far as I'm concerned anyway) to change the text instead. "Creating branch push-abc123 for revision abc123" doesn't seem much more useful than just "Creating branch push-abc123". I'm not sure what to write instead. Is just "Creating branch push-abc123" enough?

Reply:
Sure, thanks! For better or worse, I returned the message to what it was.